### PR TITLE
Add `jax_logging_level` to common config options in docs

### PR DESCRIPTION
**Pull request by skye**
Also some minor formatting changes

### CODE REVIEW

**Comment by skye** [comment]
Dumb q: how do I look at the rendered RTD docs? The action has link https://app.readthedocs.org/dashboard/jax/version/28034/edit/ which is 404 for me

**Comment by ZacCranko** [comment]
LGTM

**Review by ZacCranko — APPROVED** [review]

**Comment by ZacCranko** [comment]
> Dumb q: how do I look at the rendered RTD docs? The action has link https://app.readthedocs.org/dashboard/jax/version/28034/edit/ which is 404 for me

some more info here about the deps you need [here](https://docs.jax.dev/en/latest/developer.html#update-documentation), but the command you want is 

`sphinx-build -E -v -b html -D nb_execution_mode=off -j auto docs docs/build/html docs/config_options.rst`

**Review by ZacCranko — CHANGES_REQUESTED** [review]
Ooops looks like there's a build error now 

```
/__w/jax/jax/docs/config_options.rst:57: WARNING: Bullet list ends without a blank line; unexpected unindent.
```

**Comment by yashk2810** [comment]
> how do I look at the rendered RTD docs?

On the PR, you need to add the `documentation` label for the job to trigger. This was a recent change. For local, do what Zac said!

**Review by ZacCranko — APPROVED** [review]

**Comment by ZacCranko** [comment]
Docs build looks good, ship it!
